lib/node_modules/@stdlib/math/base/special/rcbrt/test/test.js

```diff
@@ -66,13 +66,13 @@ tape( 'the function evaluates the reciprocal cube root of `x` on the interval `[
 	expected = veryLargePositive.expected;
 	x = veryLargePositive.x;
 	for ( i = 0; i < x.length; i++ ) {
-		y = rcbrt( x[i] );
-		if ( y === expected[i] ) {
-			t.equal( y, expected[i], 'x: '+x[i]+', y: '+y+', expected: '+expected[i] );
+		y = rcbrt( x[ i ] );
```

Comment:
These changes are fine. Note, however, that, for the most part, we are not militant about this. By default, yes, we use spaces. But we also allow some flexibility depending on what aspects of the code should be visually emphasized. Can describe in more detail our "philosophy", if it can be called that, during a call. But regardless, I wouldn't go overboard with going out of your way to enforce spacing, so long as a file is consistent (e.g., nothing along the lines of `[i ]` or `[ i]`, etc). Otherwise, you're likely to just create more work for yourself.

Reply:
Okay. I understood this now. I was a bit doubtful about this, that in most of our packages, we do not have spaces here, which seems completely reasonable now.